package.json

```diff
@@ -63,9 +63,10 @@
     "expect": "^1.16.0",
     "jsdom": "^8.4.1",
     "mocha": "^2.4.5",
-    "react": "^15.0.2",
+    "proptypes": "0.14.4",
```

Comment:
This is not the correct package, should be [`prop-types`](https://www.npmjs.com/package/prop-types) (v15.5.4)